docs/getting_started.rst

```diff
@@ -147,11 +146,30 @@ Automatic Installation on Windows
 
 // TODO(hoatle): add this
 
-.. _manual-installation-on-windows:
 
 Manual Installation on Windows
 ------------------------------
 
+1. Install ``git``:
+
+Head over to https://git-scm.com/downloads and download their latest release then follow the instruction, by then you will have ``Git Bash`` already installed.
```

Comment:
latest release then follow the instruction -> latest release, and follow the instruction to install git,